internal/web3ext/web3ext.go

```diff
@@ -792,6 +792,11 @@ web3._extend({
 			call: 'istanbul_removeProxy',
 			params: 1
 		}),
+		new web3._extend.Method({
```

Comment:
nit: Is there a reason why this is a method and not a property? Imo the properties are nice because you can just see them without calling a function

Reply:
No particular reason.  I can change it to a property.